examples/basic/package.json

```diff
@@ -18,7 +20,7 @@
     "react-dom": "^15.0.2",
     "react-redux": "^4.4.5",
     "redux": "^3.5.2",
-    "redux-observable": "*",
+    "redux-observable": "^0.6.0",
```

Comment:
Don't want to hardcode the version number because then every time we bump it we have to update it here too. What's listed here really doesn't matter because we alias `redux-observable` to source code in the project root here: https://github.com/redux-observable/redux-observable/blob/master/examples/basic/webpack.config.js#L25